project/Dependencies.scala

```diff
@@ -48,11 +48,9 @@ object Dependencies {
   }
 
   val resolvers = Seq(
-    Resolver.jcenterRepo,
-    "Atlassian Releases" at "https://maven.atlassian.com/public/",
-    Resolver.bintrayRepo("scalaz", "releases"),
-    "HAT Library Artifacts Snapshots" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-snapshots.hubofallthings.com",
-    "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com"
+    "Atlassian" at "https://maven.atlassian.com/public/",
+    "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com",
+    "HAT Library Artifacts Snapshots" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-snapshots.hubofallthings.com"
```

Comment:
@pperzyna it's still here. I think your last commit was about to remove this line 🤔

Reply:
@whiteshadow-gr 

I am not able to remove snapshot from HAT, because of:
https://github.com/dataswift/HAT2.0/blob/main/project/Dependencies.scala#L44

Reply:
unfortunately that one is still required because HAT depends on a snapshot build of the adjudicator library :(